### PR TITLE
Baseline setup docs

### PR DESCRIPTION
**Pull request by jaeddy**
Adding `renv` dependency management and some additional instructions to make it easier for users to run the baseline demo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
-## Building docker images
+## Using the baseline demo in RStudio
+
+### Environment setup
+
+1. Clone this repository
+
+2. Open `metadta-automation-challenge.Rproj`
+
+3. Install packages. In the RStudio console, run:
```
Comment (thomasyu888) [inline README.md]: Run....

File: renv/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+library/
+python/
```
Comment (thomasyu888) [inline renv/.gitignore]: Does the .gitignore have to live in the renv folder, or is this just to make the repo cleaner?

File: README.md (hunk 1 above)
Comment (jaeddy) [reply README.md]: Whoops. Messed up the order. Fixed!

File: renv/.gitignore (hunk 2 above)
Comment (jaeddy) [reply renv/.gitignore]: Everything in the `renv/` directory is created by the **renv** package (i.e., via `renv::init()` or `renv::snapshot()`). I'm guessing it adds the `.gitignore` to ensure only essential files for reproducing the environment are tracked.

**Comment by jaeddy** [comment]
> Interesting. What does activate.R do?

The script is generated and used by **renv** (see comment about the `renv/` directory).

**Pull request merged**